penpot_rock/rockcraft.yaml

```diff
@@ -33,6 +33,30 @@ parts:
     override-build: |
       craftctl default
       
+      git apply <<'EOF'
```

Comment:
Out of curiosity, what is this for? (Would you mind adding a comment about it?) Thank you!

Reply:
Yes, certainly. The upstream version of the Nginx entrypoint can only apply changes to config.js once. So I need to change it so that it can apply changes to config.js multiple times. Added a comment, thank you.

Reply:
Patching the upstream has a consequence license wise, Penpot is using Mozilla Public License 2.0, I don't know how much that's compatible with our Apache 2.0 license. This needs to be considered
https://github.com/penpot/penpot/blob/develop/LICENSE

Reply:
I think the modification is acceptable as long as the source code is available to users who use this image. (see Q8 and Q10 at https://www.mozilla.org/en-US/MPL/2.0/FAQ/). And *maybe* it's okay to change to apache license as it doesn't interfere with the recipients' right to source code.

One question I've never understood is about the `license` field in `rockcraft.yaml`: does it refer to the license of the `rockcraft.yaml` file itself or the license of the ROCK image? Sometimes, for example, the penpot image contains no source code but only compiled binaries. And I am unsure if licenses for source code are applicable to binaries.

Reply:
The `license` field in rockcraft represents the license of the packaged application, we should ensure it's set accordingly.

Fine for license compatibility

Reply:
Okay, the license in `rockcraft.yaml` is already `MPL-2.0`.